test/AppInfoJsObject.cpp

```diff
@@ -21,7 +21,6 @@
 TEST(AppInfoJsObjectTest, AllProperties)
 {
   AdblockPlus::AppInfo appInfo;
-  appInfo.id = "1";
```

Comment:
Nit: Could you change the numbers here so it's "1", "2", "3", "4"? One number missing inbetween seems weird. Same for the other tests.